src/AMSWorkflow/ams/orchestrator.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021-2023 Lawrence Livermore National Security, LLC and other
+# AMSLib Project Developers
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import flux
+import json
+from flux.security import SecurityContext
```

Comment:
@milroy  I get an import error. I am guessing because of running on an old flux version. Can you make the appropriate changes to support both versions?